test/plots/barPlotTests.ts

```diff
@@ -252,8 +252,27 @@ describe("Plots", () => {
             svg.remove();
           });
 
-        });
+          it("entities().position and pixelPoint() return the same point", () => {
+            let entities = barPlot.entities();
+            entities.forEach((entity) => {
+              let pixelPoint = (<any> barPlot)._pixelPoint(entity.datum, entity.index, entity.dataset);
+              assert.strictEqual(pixelPoint.x, entity.position.x, "pixelPoint and entities().position should have same x value");
+              assert.strictEqual(pixelPoint.y, entity.position.y, "pixelPoint and entities().position should have same y value");
+            });
+            svg.remove();
+          });
```

Comment:
I personally don't think this test is that useful.

`_pixelPoint` is a non-public function, which means that we are testing against pieces of the code that users will never touch.  While this may be useful if we really want to assert this, if the behavior derived from `_pixelPoint` were to work too, then we don't need to test this directly.

Also, there is some fragility here in that if `_pixelPoint` were to have its name changed, then we would get a runtime error since our compiler will not be able to catch the name change.